docs/design/agent-config-editing/spikes/model-usability/analyze.py

```
@@ -0,0 +1,138 @@
+TASK_IDS = [t.tid for t in T.TASKS]
+
+
+def classify(record):
+    """One short label per failure."""
+    if record["correct"]:
+        return "correct"
+    error = record["error"] or ""
+    if error.startswith("api:") or "failed after" in error:
+        return "harness/api error"
+    if error == "no tool call":
+        last = [a for a in record["attempts"] if a.get("no_tool_call")]
```

Flake8 (7.3.0)

[error] local variable 'last' is assigned to but never used

(F841)